webservices/common/models/base.py

```diff
@@ -58,4 +58,4 @@ def create_session(self, options):
 
 class BaseModel(db.Model):
     __abstract__ = True
-    idx = db.Column(db.Integer, primary_key=True)
+    idx = db.Column(db.Integer, primary_key=True, autoincrement=True)
```

Comment:
Based on the note I saw in your commit message, we were relying on this parameter being set automatically before with the `1.0.x` version?

Reply:
Correct!